src/tests/unit/tests/reports/package/__snapshots__/footer-text-for-service.test.tsx.snap

```diff
@@ -5,13 +5,19 @@ exports[`FooterTextForService renders: footer 1`] = `
   This automated checks result was generated using the 
   app-name
    
-  that helps find some of the most common accessibility issues. The scan was performed using 
+  that helps find some of the most common accessibility issues. The scan was
+   
+  performed in a clean browser environment, using 
   engine-name
    
   engine-version
    and 
   environmentName
-  . For a complete WCAG 2.1 compliance assessment please visit
+   
+  with a display resolution of 
+  . For a complete
```

Comment:
Should we modify `footer-text-for-service.test.tsx` to include a stub for the display resolution? I don't see any updates to this test file in this PR